uefi-macros/src/lib.rs

```diff
@@ -211,3 +211,39 @@ pub fn entry(args: TokenStream, input: TokenStream) -> TokenStream {
     };
     result.into()
 }
+
+#[proc_macro]
```

Comment:
Please add a `///` docstring here that briefly explains what it does and how to use it. Something like:

```rust
/// Create a `CStr8` from a string.
///
/// # Example
///
/// ```
/// use uefi_macros::cstr8;
/// assert_eq!(cstr8!("abc").to_bytes_with_nul(), [97, 98, 99, 0]);
/// ```
```